xicam/plugins/plugin.py

```diff
@@ -1,5 +1,7 @@
 class PluginType:
     is_singleton = False
+    entrypoint_prefix = 'xicam.plugins.'
```

Comment:
also i like having the default entrypoint prefix directed to 'xicam.plugins', and it can easily be set to a custom value (like you did with 'databroker.').